src/main/groovy/net/researchgate/release/tasks/PreTagCommit.groovy

```diff
@@ -22,8 +22,8 @@ class PreTagCommit extends BaseReleaseTask {
         if (projectAttributes.usesSnapshot || projectAttributes.versionModified || projectAttributes.propertiesFileCreated) {
             // should only be committed if the project was using a snapshot version.
             String message = extension.preTagCommitMessage.get() + " '${tagName()}'."
-            if (extension.preCommitText) {
-                message = "${extension.preCommitText.get()} ${message}"
+            if (extension.preCommitText.get()) {
```

Comment:
This is the only line in the PR that I would second, as it also is the same I did in #384 not seeing this PR. :-D

Reply:
Which was merged just now, so my recommendation would be to close this PR unmerged. :-)